t/rose-metadata-graph/01-data.t

```diff
@@ -26,6 +26,45 @@ python -c "import pygraphviz" 2>/dev/null || \
 #-------------------------------------------------------------------------------
 tests 12
 #-------------------------------------------------------------------------------
+# Filter graphviz output.
+function filter_graphviz() {
```

Comment:
As it's used in more than one test, do you want to move this out to the test_header file? (Could prove useful elsewhere in future as well as meaning you only have to remember to edit the one place rather than 2)